<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stuInfoDialog</class>
 <widget class="QDialog" name="stuInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>278</y>
     <width>101</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>其他：</string>
   </property>
  </widget>
  <widget class="QLabel" name="otherLabel">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>278</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>其他：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>101</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>兴趣：</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>191</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>40</y>
     <width>96</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>87</y>
     <width>96</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>87</y>
     <width>251</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>134</y>
     <width>96</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLabel" name="sexLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>134</y>
     <width>211</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>181</y>
     <width>96</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>年龄：</string>
   </property>
  </widget>
  <widget class="QLabel" name="ageLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>181</y>
     <width>211</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>年龄</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>228</y>
     <width>96</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>专业：</string>
   </property>
  </widget>
  <widget class="QLabel" name="departmentLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>228</y>
     <width>281</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="inteTableWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>兴趣类</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>兴趣</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="addInteButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>660</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加兴趣(&amp;A)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delInteButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>660</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除兴趣(&amp;P)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改个人信息(&amp;S)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
